src/Avalonia.Controls/TopLevel.cs

```diff
@@ -350,6 +350,12 @@ PixelPoint IRenderRoot.PointToScreen(Point p)
         /// </summary>
         protected virtual ILayoutManager CreateLayoutManager() => new LayoutManager(this);
 
+        public override void InvalidateFlowDirection()
```

Comment:
This can be removed

Reply:
All `TopLevel` should not get RenderTransform at all.

Reply:
Git it